### ISSUE TITLE
Include links to Landing Page sections on Build Analysis

### ISSUE THREAD

**Issue by missymessa** [dotnet/arcade#10964]
Keeping in mind that we do have a character limit on the check, we should find some way to introduce some kind of help information on the check in the form of a low-tech tooltip (e.g. alt text over an image?) or something similar (not something that would add more noise to the check) if possible. Questions that these tooltips could clear up are: 

- What's the "Report Infrastructure Issue" (Known Issues) link for? When do I use reporting infra versus product? 
- What data is the test failure rate from AzDO based on? 
- (will add more :) )

**Comment by ChadNedzlek** [comment]
Is there some way to do this? I can't see any way to actually get any sort of tooltips to work. Also: alt-text for images is not ok from an accessibility point of view (even if we could figure out how to make that work). A screen reader treats the text as an alternative to the image and will read it aloud when encountered... so it should describe the image, and no more (it should be an alternative to the image itself). Using them as fake UI violated accessibility guidelines.

I do think we probably need a little help section/icon that links to some helpful page that can answer questions though.

It's unfortunate how limited we are in our UI options because of markdown.

**Comment by missymessa** [comment]
We'll include links to the Landing Page for better context around features.

**Comment by AlitzelMendez** [comment]
add link to landing page in https://github.com/dotnet/arcade/blob/main/Documentation/Projects/Build%20Analysis/LandingPage.md

**Comment by AlitzelMendez** [comment]
PR: https://github.com/dotnet/arcade/pull/12614
